contracts/ronin/gateway/RoninGatewayV2.sol

```diff
@@ -52,15 +48,15 @@ contract RoninGatewayV2 is
   /// @dev Mapping from token address => chain id => mainchain token address
   mapping(address => mapping(uint256 => MappedToken)) internal _mainchainToken;
 
-  /// @dev The ronin validator contract
+  /// @notice deprecated
   IRoninValidatorSet internal _validatorContract;
```

Comment:
Please rename the deprecated values as `____deprecated<number_if_colliding>` to avoid accidental misuse, and comment with `@custom:deprecated` tag as NatSpec syntax.

```suggestion
  /// @custom:deprecated previously `_validatorContract` (non-zero value)
  address private ____deprecated0;
```

Reply:
updated.